src/components/AccountFormContainer/index.tsx

```diff
@@ -41,9 +41,11 @@ const AccountFormContainer = () => {
       method="post"
       action="/library-card/api/submit"
     >
-      <AccountFormFields />
-
-      <AcceptTermsFormFields />
+      <fieldset>
+        <legend>Account form fields</legend>
```

Comment:
Wrapping fields that belong together in a `fieldset` and `legend`. The `legend` is visually hidden.